script.js

```diff
@@ -0,0 +1,158 @@
+        this.domElement.classList.add('button__'.concat(name.toLowerCase()));
+        this.domElement.setAttribute('id', name);
+        this.addToKeyboard(keyboard);
+        this.keyBoard = keyboard;
+        this.spanText = value;
+        this.changeText();
+        if (name === 'ControlLeft' || name === 'ControlRight' || name === 'ShiftLeft' || name === 'ShiftRight' || name === 'AltLeft' || name === 'AltRight' || name === 'ArrowUp' || name === 'ArrowLeft' || name === 'ArrowRight' || name === 'ArrowDown' || name === 'CapsLock' || name === 'OSLeft') {
+            this.inputTypeValue = false;
+            this.workerType = true;
+        } else if (name === 'Space') {
+            this.value = ' ';
+            this.workerType = true;
+        } else if (name === 'Enter') {
+            this.value = '\n';
+            this.workerType = true;
+        } else if (name === 'Backspace') {
+            this.value = 'backspace';
+            this.workerType = true;
+        } else if (name === 'Delete') {
+            this.value = 'delete';
+            this.workerType = true;
+        } else if (name === 'Tab') {
+            this.value = '\t';
+            this.workerType = true;
+        }
+        if (this.workerType) this.domElement.classList.add('worker-type');
+        this.locked = false;
+    }
+
+    addToKeyboard(keyBoard) {
+        keyBoard.addBtn(this);
+    }
+
+    // View methods
+    changeText(up) {
+        const spanValue = this.spanForValue;
+        const spanShiftValue = this.spanForShiftValue;
+        if (Array.isArray(this.value) && this.multiLang) {
+            if (this.keyBoard.language === 'en') [spanValue.innerHTML, spanShiftValue.innerHTML] = this.value;
+            else if (this.keyBoard.language === 'ru') {
+                [spanValue.innerHTML, spanShiftValue.innerHTML] = this.altValue;
+                if (up) spanValue.innerHTML = String(spanValue.innerHTML).toUpperCase();
+            }
+        } else if (!Array.isArray(this.value) && this.multiLang) {
+            if (this.keyBoard.language === 'en') spanValue.innerHTML = up ? String(this.value).toUpperCase() : this.value;
+            else if (this.keyBoard.language === 'ru') spanValue.innerHTML = up ? String(this.altValue).toUpperCase() : this.altValue;
+        } else spanValue.innerHTML = this.spanText;
+    }
+
+    addClass(className) {
+        this.domElement.classList.add(className);
+    }
+
+    removeClass(className) {
+        this.domElement.classList.remove(className);
+    }
+}
+
+// Creating buttons
+btns.Backquote = new Button('Backquote', ['`', '~'], ['ё', ''], true);
+for (let i = 1; i <= 10; i += 1) {
+    const symb = [['!', '!'], ['@', '"'], ['#', '№'], ['$', ';'], ['%', '%'], ['^', ':'], ['&', '?'], ['*', '*'], ['(', '('], [')', ')']];
+    // digits
+    const num = i < 10 ? i : 0;
+    const btn = 'Digit'.concat(num);
+    btns[btn] = new Button(btn, [num, symb[i - 1][0]], [num, symb[i - 1][1]], true);
```

Comment:
Can it be more declarative rather than imperative? I want to see what buttons are look like and not variables manipulations